projects/dxc-ngx-cdk/src/lib/dxc-lookup/orghlookup/dxc-orghlookup.component.html

```diff
@@ -23,9 +23,9 @@ <h6 class="modal-title ml-2">
           </span>
         </dxc-button-icon>
       </dxc-button>
-      <div class="float-md-right">
+      <div class="float-md-right d-none d-sm-block d-md-inline-block">
```

Comment:
why need css class even ngIf is defined on buttons

Reply:
because we have to remove treeview button and gridview button in case of mobile view.